### PR TITLE
Start replacing Goobi by Kitodo

### PR DESCRIPTION
**Pull request by stweil**
* s/www.goobi.org/www.kitodo.org/
* s/Goobi Community/Kitodo Community/
* s/goobi is loading/Kitodo is loading/

Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

**Comment by stweil** [comment]
@henning-gerhardt, this modifications are of course far from being complete. I wanted them for an experimental docker container with _Kitodo.Production_. Should we add an issue for the Goobi-Kitodo rename, so we can synchronize our activities there? One of my next needs is the Kitodo logo in different sizes (or maybe as a SVG file).

**Comment by henning-gerhardt** [comment]
Until now there is no final plan for renaming process from Goobi to Kitodo and how this will be done. This need some days as parts of release management are in or will be in vacation for next two weeks.

**Comment by stweil** [comment]
A first experimental [Docker container](https://hub.docker.com/r/kitodo/production/) is now available.

File: Goobi/src/de/sub/goobi/forms/HelperForm.java [hunk 1]
```diff
@@ -3,13 +3,6 @@
...
  * This file is part of the Goobi Application - a Workflow tool for the support of mass digitization.
  * 
- * Visit the websites for more information. 
- *     		- http://www.kitodo.org
- *     		- https://github.com/goobi/goobi-production
- * 		    - http://gdz.sub.uni-goettingen.de
- * 			- http://www.intranda.com
- * 			- http://digiverso.com 
- * 
  * This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free
```
Comment (henning-gerhardt) [inline Goobi/src/de/sub/goobi/forms/HelperForm.java]: Removing this line are intended ?

File: Goobi/src/de/sub/goobi/forms/HelperForm.java [hunk 2]
```diff
@@ -18,14 +11,6 @@
...
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- * 
- * Linking this library statically or dynamically with other modules is making a combined work based on this library. Thus, the terms and conditions
- * of the GNU General Public License cover the whole combination. As a special exception, the copyright holders of this library give you permission to
- * link this library with independent modules to produce an executable, regardless of the license terms of these independent modules, and to copy and
- * distribute the resulting executable under terms of your choice, provided that you also meet, for each linked independent module, the terms and
- * conditions of the license of that module. An independent module is a module which is not derived from or based on this library. If you modify this
- * library, you may extend this exception to your version of the library, but you are not obliged to do so. If you do not wish to do so, delete this
- * exception statement from your version.
  */
```
Comment (henning-gerhardt) [inline Goobi/src/de/sub/goobi/forms/HelperForm.java]: Removing this lines are intended ?

File: Goobi/src/de/sub/goobi/forms/HelperForm.java (hunk 1 above)
Comment (stweil) [inline Goobi/src/de/sub/goobi/forms/HelperForm.java]: These lines contain several whitespace issues (tabs/blank, spaces at line endings) which are nagging when editing the file.

As the headers will need a major rework anyway, I started with removing unnecessary parts.

File: Goobi/src/de/sub/goobi/forms/HelperForm.java (hunk 2 above)
Comment (stweil) [inline Goobi/src/de/sub/goobi/forms/HelperForm.java]: Yes. Do you think we'll still need it for Kitodo?

File: Goobi/src/de/sub/goobi/forms/HelperForm.java (hunk 1 above)
Comment (henning-gerhardt) [inline Goobi/src/de/sub/goobi/forms/HelperForm.java]: This should be done everywhere on header rework on license change.

File: Goobi/src/de/sub/goobi/forms/HelperForm.java (hunk 2 above)
Comment (henning-gerhardt) [inline Goobi/src/de/sub/goobi/forms/HelperForm.java]: Header rework containing a new license section so this change is now unnecessary.

**Comment by stweil** [comment]
I restored the header comments in Goobi/src/de/sub/goobi/forms/HelperForm.java now.

**Pull request merged**